pages/content/amp-dev/documentation/guides-and-tutorials/start/bento_guide.md

```diff
@@ -0,0 +1,388 @@
+            layout="intrinsic"
+          ></amp-img>
+        </article>
+        <article class="article-slide">
+          <div class="article-description">
+            <a href="https://www.npr.org/sections/national/">
+              <h2>National</h2>
+            </a>
+            <a
+              href="https://www.npr.org/2020/12/03/942034617/time-names-its-kid-of-the-year-water-testing-scientist-gitanjali-rao"
+            >
+              <amp-fit-text height="70" min-font-size="20" max-font-size="20">
+                'Time' Names Its Kid Of The Year: Water-Testing Scientist
+                Gitanjali Rao
+              </amp-fit-text>
+            </a>
+          </div>
+          <amp-img
+            class="article-img"
+            width="700"
+            height="393"
+            src="https://media.npr.org/assets/img/2020/12/03/time201207_koty.coverfinal_wide-a7cff22dc41f55823d394817f2c1a0acfc105a66.jpg?s=600"
+            layout="intrinsic"
+          ></amp-img>
+        </article>
+        <article class="article-slide">
+          <div class="article-description">
+            <a href="https://www.npr.org/sections/book-reviews/">
+              <h2>Book Reviews</h2>
+            </a>
+            <a
+              href="https://www.npr.org/2020/12/04/941982629/maureen-corrigans-10-books-that-will-connect-you-in-a-socially-distant-year"
+            >
+              <amp-fit-text height="70" min-font-size="20" max-font-size="20">
+                Maureen Corrigan's 10 Books That Will Connect You In A Socially
+                Distant Year
+              </amp-fit-text>
+            </a>
+          </div>
+          <amp-img
+            class="article-img"
+            width="700"
+            height="393"
+            src="https://media.npr.org/assets/img/2020/12/03/corrigan2020_wide-26e0b93fd657db854e1ba2bdb7b18539ca72ecfa-s1100-c85.jpg"
+            layout="intrinsic"
+          ></amp-img>
+        </article>
+      </amp-base-carousel>
+    </div>
+  </body>
+</html>
+```
+
+See the [demo in action on Glitch](TODO).
+
+# Working with experiments
+
+Bento AMP is in experimental mode and available through the developer preview. The AMP team welcomes developer feedback through [GitHub](https://github.com/ampproject/amphtml/issues) and our [Slack](https://docs.google.com/forms/d/e/1FAIpQLSd83J2IZA6cdR6jPwABGsJE8YL4pkypAbKMGgUZZriU7Qu6Tg/viewform?fbzx=4406980310789882877) channel. Please reach out with any questions or issues. 
+
+
+## Page experience
+
+Bento components are designed to be highly performant and contribute to an excellent page experience. Developers are highly encouraged to [file issues](https://github.com/ampproject/amphtml/issues) if they see contradictory results.  
+
+
+## AMP caches and validation
+
+Using Bento components during the limited developer preview makes your web page an invalid AMP page. Therefore, your doc and any Bento components are not added to any AMP caches.
```

Comment:
Are these intended to be subheadings of `Working with experiments`?

Reply:
good catch - fixed!